spec/controllers/photos_controller_spec.rb

```diff
@@ -54,6 +54,12 @@ def valid_session
       assigns(:type).should eq "garden"
```

Comment:
Controller and view specs are BOTH deprecated? Well it doesn't make sense to me that this get moved to...model?... spec without also moving all the rest of it, because splitting up a test that's done almost identically (but with different input) several times seems really disorganized.